icomoon/demo-files/demo.js

```diff
@@ -0,0 +1,30 @@
+if (!('boxShadow' in document.body.style)) {
+	document.body.setAttribute('class', 'noBoxShadow');
+}
+
+document.body.addEventListener("click", function(e) {
+	var target = e.target;
+	if (target.tagName === "INPUT" &&
+		target.getAttribute('class').indexOf('liga') === -1) {
+		target.select();
+	}
+});
+
+(function() {
+	var fontSize = document.getElementById('fontSize'),
```

Comment:
I agree with the "best practices We disagree with" section in this case. This way to declare variables could cause problems if we miss a coma for instance.

I would declare three different vars